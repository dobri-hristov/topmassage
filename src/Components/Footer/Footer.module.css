.container {
    display: flex;
    min-height: 60px;
    color: black;
    background-color: #757594;
    padding-top: 12px;
}

.info {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-between;
    align-items: center;
    width: 80%;
    margin: 0 auto;
}

.copyright {
    /* order: 2 */
}

.contacts {
    display: flex;
    flex-wrap: wrap;
    list-style: none;
    padding: 0;
    margin: 0;
    /* order: 1 */
}

.contacts li {
    text-decoration: none;
    margin: 5px 10px;
    border-radius: 50%;
    height: 40px;
    width: 40px;
    text-align: center;
    background: rgb(231, 226, 226);
    box-shadow: 0px 5px 10px 0px rgba(0, 0, 0, 0.75);
}

.contacts li i {
    font-size: 20px;
    margin: 10px auto;
}

.facebook:hover {
    background: blue;
    color: white;
}

.insta:hover {
    background: #f09433;
    background: -moz-linear-gradient(45deg, #f09433 0%, #e6683c 25%, #dc2743 50%, #cc2366 75%, #bc1888 100%);
    background: -webkit-linear-gradient(45deg, #f09433 0%, #e6683c 25%, #dc2743 50%, #cc2366 75%, #bc1888 100%);
    background: linear-gradient(45deg, #f09433 0%, #e6683c 25%, #dc2743 50%, #cc2366 75%, #bc1888 100%);
    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#f09433', endColorstr='#bc1888', GradientType=1);
    color: white;
}

.youtube:hover {
    background: red;
    color: white;
}

@media screen and (max-width: 600px) {
    .info {
        width: 90%;
        align-items: center;
        flex-flow: column-reverse;
        text-align: center;
    }

}

@media screen and (max-width: 250px) {
    .contacts {
        flex-flow: column;
    }

    .contacts li {
        margin-bottom: 20px;
        height: 70px;
        width: 70px;
    }

    .contacts li i {
        font-size: 30px;
        margin: 20px auto;
    }
}