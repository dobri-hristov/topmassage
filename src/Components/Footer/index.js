import React from 'react'
import styles from './Footer.module.css'

const Footer = () => {
    const date = new Date()
    const year = date.getFullYear()

    return (
        <div className={styles.container}>
            <div className={styles.info}>
                <p className={styles.copyright}> &#169; {year} Top Massage. All rights reserved</p>
                <ul className={styles.contacts}>
                    <li
                        className={styles.facebook}>
                        <i className="fab fa-facebook-f"></i>
                    </li>
                    <li
                        className={styles.insta}>
                        <i className="fab fa-instagram"></i>
                    </li>
                    <li
                        className={styles.youtube}>
                        <i className="fab fa-youtube"></i>
                    </li>
                </ul>
            </div>
        </div>
    )
}

export default Footer
