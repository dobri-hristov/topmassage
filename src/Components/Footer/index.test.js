import React from 'react'
import TestRenderer from 'react-test-renderer'
import Footer from './index.js'

describe('Footer component', () => {
    test('should render correctly', () => {
        const component = TestRenderer.create(
            <Footer />
        )
        const tree = component.toJSON()
        expect(tree).toMatchSnapshot()
    })
})