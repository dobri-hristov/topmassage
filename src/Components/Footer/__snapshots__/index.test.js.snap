// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render correctly 1`] = `
<div
  className="container"
>
  <div
    className="info"
  >
    <p
      className="copyright"
    >
       Â© 
      2022
       Top Massage. All rights reserved
    </p>
    <ul
      className="contacts"
    >
      <li
        className="facebook"
      >
        <i
          className="fab fa-facebook-f"
        />
      </li>
      <li
        className="insta"
      >
        <i
          className="fab fa-instagram"
        />
      </li>
      <li
        className="youtube"
      >
        <i
          className="fab fa-youtube"
        />
      </li>
    </ul>
  </div>
</div>
`;
